layers/+tools/lsp/funcs.el

```diff
@@ -38,36 +38,44 @@ https://github.com/emacs-lsp/lsp-javascript/issues/9#issuecomment-379515379"
       (spacemacs//lsp-bind-peek-navigation-functions "G")))
 
   (spacemacs/set-leader-keys-for-minor-mode 'lsp-mode
-    ;;format
+    ;; format
     "=b" #'lsp-format-buffer
-    ;;goto
+    "=r" #'lsp-format-region
+    ;; goto
     "gt" #'lsp-find-type-definition
     "gk" #'spacemacs/lsp-avy-goto-word
     "gK" #'spacemacs/lsp-avy-goto-symbol
-    "ge" #'lsp-ui-flycheck-list
     "gM" #'lsp-ui-imenu
-    ;;help
+    ;; help
     "hh" #'lsp-describe-thing-at-point
-    ;;jump
-    ;;backend
+    ;; jump
+    ;; backend
     "ba" #'lsp-execute-code-action
     "bd" #'lsp-describe-session
     "br" #'lsp-restart-workspace
-    ;;refactor
+    "bs" #'lsp-shutdown-workspace
+    ;; refactor
     "rr" #'lsp-rename
-    ;;toggles
+    ;; toggles
     "Td" #'lsp-ui-doc-mode
     "Ts" #'lsp-ui-sideline-mode
     "TF" #'spacemacs/lsp-ui-doc-func
     "TS" #'spacemacs/lsp-ui-sideline-symb
-    "TI" #'spacemacs/lsp-ui-sideline-ignore-duplicate))
+    "TI" #'spacemacs/lsp-ui-sideline-ignore-duplicate
+    "Tl" #'lsp-lens-mode
+    ;; folders
+    "fs" #'lsp-workspace-folders-switch
```

Comment:
These keybindings conflict with `Web frameworks` conventions (`SPC m f`). Can you think of any other prefix that works?

Reply:
`SPC m b f a` or `SPC m w f a` or `SPC m F a`?

Reply:
`SPC m F` should work as there is only on [wrongly] defined keybinding for Haskell that conflicts with it (which should be under `SPC m =` as it's a formatting one).

Reply:
Force pushed a version with the new keybings.